public/images/account/org-benefits/2.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
```

Comment:
If you'd like to resize the icon in CSS, you may remove hardcoded width & height from the SVG tag.